Auto_Crafters.md (#51)
StarLegacy/StarLegacy.Wiki: Machines/Auto_Crafters.md

# Auto Crafters

> Auto crafters are an automated way to get your crafting needs met (duh). 

Reviewer: change to something more sensible
```suggestion
Auto crafters are machines used to mass produce a specific crafting recipe using input and output chests,
crafting the recipe given in the recipe holder dropper/dispenser block.
```

> There are 3 different tiers from 1-3, each tier they craft much faster but also use more energy. 

Reviewer: ```suggestion
Auto Crafters come in 3 tiers. Higher tiers craft items faster (thus using more power).
```

> - Tier 1 costs 5k

Reviewer: ```suggestion
- **Tier 1** (Iron) (2 crafts/second) (200,000 power capacity)
```

> - Tier 2 10k,

Reviewer: ```suggestion
- **Tier 2** (Gold) (4 crafts/second) (400,000 power capacity)
```

> - Tier 3 20k. 

Reviewer: ```suggestion
- **Tier 3** (Diamond) (6 crafts/second) (600,000 power capacity)
```

> /advance them all to auto craft what ever you want!

Reviewer: ```suggestion
To unlock them, use i.e. `/advance AUTO_CRAFTER_ONE`. They cost C5,000, C10,000, and C20,000.
```

> ## To Use Auto Crafters

Reviewer: ```suggestion
## Usage
```

> - Make sure the sign on the auto crafter says [AutoCrafter]

Reviewer: ```suggestion
- Put `[autocrafter]` on the first line of the sign
```

> - The Input is the dropper on top of the glass, left of the sign.

Reviewer: ```suggestion
- The input container is the container on the left side of the multiblock, atop the glass block.
```

> - Output is the dropper on the other side on top of the crafting table.

Reviewer: ```suggestion
- The output container is the container on the right side of the multiblock, atop the crafting table.
```

> - Using items form the recipe in the middle dropper as if it were a crafting table.

Reviewer: ```suggestion
- In the dropper/dispenser in the center, place the recipe as if you were crafting the item normally.
```

> ## To Make an Auto Crafter

Reviewer: ```suggestion
## Multiblock
```

> To build the auto crafter you will need:

Reviewer: ```suggestion
### Requirements
```

> 8 Tier blocks, (Iron, Gold, or Diamond) 8 Glass Panes, 

Reviewer: ```suggestion
* 8x tier block (iron, gold, or diamond)
* 8x glass pane
```

> 3 Glass blocks, 2 Titanium blocks, 3 droppers, 1 noteblock, 

Reviewer: ```suggestion
* 3x glass block
* 2x titanium block
* 3x dropper
* 1x wire input computer (note block) 
```

> 1 furnace (with 1 focusing lens in each slot), and 2 wire/end rods

Reviewer: ```suggestion
* 1x furnace
* 2x wire (end rod)
```
